poc/test/rte-implementation-v1alpha1/main.go

```diff
@@ -15,8 +15,7 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/certwatcher"
 
-	runtimev1 "sigs.k8s.io/cluster-api/exp/runtime/api/v1beta1"
-	"sigs.k8s.io/cluster-api/exp/runtime/hooks/api/v1alpha1"
+	runtimev1alpha1 "sigs.k8s.io/cluster-api/exp/runtime/hooks/api/v1alpha1"
```

Comment:
Can you please add this config to .golangci.yml and adjust the imports accordingly?
```yaml
      - pkg: sigs.k8s.io/cluster-api/exp/runtime/hooks/api/v1alpha1
        alias: runtimehooksv1
```

Context: we are using runtimev1 for the package with ExtensionConfig, v1 vs v1alpha1 we usually use v1 as the latest v1x version, something like runtimev1alpha1 would only be used if we have to access multiple versions (e.g. clusterv1, clusterv1alpha3, clusterv1alpha4)